test/refactoring/ConvertAsync/convert_function.swift

```diff
@@ -287,15 +287,12 @@ func testReturnHandling2(completion: @escaping (String) -> ()) {
 // RETURN-HANDLING2-NEXT:   }
 // RETURN-HANDLING2-NEXT: }
 
-// FIXME: We should arguably be able to handle transforming this completion handler call (rdar://78011350).
-// RUN: %refactor -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix=RETURN-HANDLING3 %s
+// RUN: %refactor-check-compiles -add-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 | %FileCheck -check-prefix=RETURN-HANDLING3 %s
 func testReturnHandling3(_ completion: (String?, Error?) -> Void) {
   return (completion("", nil))
 }
 // RETURN-HANDLING3:      func testReturnHandling3() async throws -> String {
-// RETURN-HANDLING3-NEXT:   return try await withCheckedThrowingContinuation { continuation in  
-// RETURN-HANDLING3-NEXT:     (continuation.resume(returning: ""))
-// RETURN-HANDLING3-NEXT:   }
+// RETURN-HANDLING3-NEXT:   return ""
```

Comment:
Could you add a `{{^}}` to this check line just to make sure we don't accidentally insert a double return?

Reply:
Sure.